src/Inciter/FluxCorrector.C

```diff
@@ -396,8 +396,8 @@ FluxCorrector::lim( const std::vector< std::size_t >& inpoel,
 //! \param[in] Ul Low order solution
 //! \param[inout] Q The maximum and mimimum unknowns of elements surrounding
 //!   each node
-//! \param[in,out] Limited antidiffusive element contributions scatter-added to
-//!   nodes
+//! \param[in,out] A Limited antidiffusive element contributions scatter-added
```

Comment:
This is not a fortran but a doxygen comment. I believe in the past one was required to use `inout` then they switched the syntax to `in,out`, then now doxygen, I believe, understands both. But will put in a comma so it is consistent.